### ISSUE TITLE
Lonely back tick gets escaped and un-escaped on successive formats

### ISSUE THREAD

**Issue by jfmengels** [avh4/elm-format#681]
## Problem

When parsing Markdown in comments, if there is a back tick without a corresponding opening/closing back tick, then `elm-format` will re-format it to be escaped. When run again, `elm-format` will un-escape the back tick.

I noticed this because `--validate` noticed a problem, and kept reporting one even after running `elm-format` several times.
Unfortunately, running with `--validate` didn't tell me what the problem was, so it was confusing (but fortunately, I hadn't touched much code so I could figure the problem quickly).

## SSCCE

```elm
module A exposing (..)

{-| [foo`]
-}


a =
    1
```

:arrow_down: :arrow_up: 

```elm
module A exposing (..)

{-| [foo\`]
-}


a =
    1
```

## Additional notes

I imagine that the solution here is to have `elm-format` choose what it wants the result to be (escaped or not), and stick to it.

**Comment by avh4** [comment]
Looks like this is maybe related to https://github.com/avh4/elm-format/issues/432 and I just need to add "`" to the list of special formatting characters that should always be escaped.

**Comment by avh4** [comment]
Should be fixed in 0.8.4 via https://github.com/avh4/elm-format/pull/684
